tests/testproject/TestProject.IntegrationServiceA/TestProject.IntegrationServiceA.csproj

```diff
@@ -32,6 +32,7 @@
     <ProjectReference Condition="'$(TestsRunningOutsideOfRepo)' != 'true'" Include="@(ComponentReferenceForTests -> '$(RepoRoot)src\Components\%(Identity)\%(Identity).csproj')" />
     <PackageReference Condition="'$(TestsRunningOutsideOfRepo)' == 'true'" Include="@(ComponentReferenceForTests)" />
     <PackageReference Include="Polly" />
```

Comment:
```suggestion
```

We can remove this now that we are using Http.Resilience.

Actually, we shouldn't be using Http.Resilience, that is for Http calls. We should just need to change this to `Polly.Core`.